extensions/objects/templates/music-phrase-editor.component.ts

```diff
@@ -86,10 +86,6 @@ export class MusicPhraseEditorComponent implements OnInit {
 
   private _createProxy(): void {
     this._proxy = new Proxy(this._localValue, {
-      apply: function(target, thisArg, argumentsList) {
-        return thisArg[
-          target as unknown as string].apply(this, argumentsList);
-      },
```

Comment:
This part of the code does not execute because `this._localValue` must be a callable itself. That is, it must be a function object. But in this case, it is a variable.
Source: [developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/Proxy/apply#invariants)